sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/_generated/_azure_digital_twins_api.py

```diff
@@ -39,6 +39,7 @@ class AzureDigitalTwinsAPI(object):
     :param credential: Credential needed for the client to connect to Azure.
     :type credential: ~azure.core.credentials.TokenCredential
     :param str base_url: Service URL
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no Retry-After header is present.
```

Comment:
I'm a little suspicious of this one - where does this come from? The new swagger has no changes except for the API version.